src/plugins/label/styles.scss

```diff
@@ -0,0 +1,29 @@
+.ExpandableTestWidget {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    flex-grow: 1;
+    align-self: stretch;
+    // background: var(--accent);
+    cursor: pointer;
+}
+
+.label {
+    font-size: 3rem;
+}
+
+.label_span {
+    font-style: italic;
+    color: rgba(220, 220, 220, 0.75)
```

Comment:
We shouldn't use hard coded colors, as this will break on different/custom themes, instead it's better to use pre-defined css variables. I think here `var(--text-subtle-1)` or `var(--text-subtle-2)` will work well